release/README.md

```diff
@@ -57,7 +57,7 @@ This script does the following:
 In the event of any of the steps above failing you might have to revert the repository to its original state. Follow the steps below as required:
```

Comment:
Can't comment on those lines directly since they're too far from the diff, but I see references of `master` on line 8.